src/lib/routes/client-api/feature.test.ts

```diff
@@ -156,13 +158,16 @@ test('fetch single feature', async () => {
         strategies: [{ name: 'default' }],
     });
 
-    return request
-        .get(`${base}/api/client/features/test_`)
-        .expect('Content-Type', /json/)
-        .expect(200)
-        .expect((res) => {
-            expect(res.body.name === 'test_').toBe(true);
-        });
+    return (
+        request
+            .get(`${base}/api/client/features/test_`)
+            .expect('Content-Type', /json/)
+            // .expect(200)
```

Comment:
Should comment it in or remove

Reply:
this PR was not ready when you started commenting. It's removed now :)